mscore/scoreview.cpp

```diff
@@ -4882,6 +4881,10 @@ void ScoreView::cmdCreateTuplet( ChordRest* cr, Tuplet* tuplet)
                   sm->postEvent(new CommandEvent("note-input"));
             _score->inputState().setDuration(tuplet->baseLen());
             }
+#ifndef NDEBUG
```

Comment:
Copied from https://github.com/musescore/MuseScore/blob/master/libmscore/layout.cpp#L3580 - should we prefer to do it in all builds in every case?